src/app/copy/block-copy/block-copy.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  ViewEncapsulation
+} from '@angular/core';
+
+import { CopyService } from '../copy-service/copy.service';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'pfng-block-copy',
+  templateUrl: './block-copy.component.html',
+  styleUrls: ['./block-copy.component.less']
+})
+
+export class BlockCopyComponent implements OnInit {
+  @Input('label') label: string;
+  @Input('token') token: string = 'Missing \'token\' @Input property';
+  @Input('copyBtnTxt') copyBtnTxt: string = 'Copy';
+  @Input('tokenPanelOpen') tokenPanelOpen: boolean = false;
```

Comment:
I'd like to rename this as `expanded` or possibly `panelExpanded`, This would be more consistent with other components.